spec/lib/danger/ci_sources/jenkins_spec.rb

```diff
@@ -86,7 +86,7 @@
         expect(described_class.validates_as_ci?(valid_env)).to be true
       end
 
-      it "doesnt validate when require env variables are not set" do
+      it "doesn't validate when require env variables are not set" do
```

Comment:
Do we have a style guide or anything establishing this? I saw a few different misspellings of doesn't/does not so I wasn't sure which to use.